### ISSUE TITLE
Documentation does not mention the new classes BsonDataWriter and BsonDataReader

### ISSUE THREAD

**Issue by Nikkelmann** [JamesNK/Newtonsoft.Json.Bson#11]
The [documentation for Bson](https://www.newtonsoft.com/json/help/html/N_Newtonsoft_Json_Bson.htm) does not mention the new classes, only that Bson has moved to a new package.

[Issue 1277](https://github.com/JamesNK/Newtonsoft.Json/issues/1277) over at Newtonsoft.Json helped me in migrating from `BsonWriter` and `BsonReader` to the new `BsonDataWriter` and `BsonDataReader`.

**Comment by ppumkin** [comment]
https://www.newtonsoft.com/json/help/html/SerializeToBson.htm

**Comment by DaveShopCtrl** [comment]
Documentation is still not updated..

**Comment by markholst** [comment]
Yes, please, update the documentation!  Or, update the Obsolete message to specify we should use the BsonDataReader/BsonDataWriter equivalents.

I spent about an hour trying to get rid of these warnings, just to realise it was a documentation issue.

I don't want to sound ungrateful by the way.  Awesome packages.

**Comment by majew7** [comment]
Unfortunately, documentation is still not updated in early 2024.

So grateful to learn that the obsolete solution is two parts:
1) Change to `Newtonsoft.Json.Bson` package and namespace.
2) Replace `BsonReader` with `BsonDataReader`
